### PR TITLE
Add Qingping integration

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

Add Qingping integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/76598
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/3582
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/qingping.markdown [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Integrates [Qingping](https://qingping.co/) devices into Home Assistant.
+
+{% include integrations/config_flow.md %}
+
+The Qingping integration will automatically discover devices once the [Bluetooth](/integrations/bluetooth) integration is enabled and functional.
+
+## Supported devices
+
+- [Motion & Ambient Light Sensor](https://www.qingping.co/motion-light-sensor/overview)
```
Comment (bdraco) [inline source/_integrations/qingping.markdown]: This is the only one that has showed up from aliexpress.  I'll add more when they show

File: source/_integrations/qingping.markdown [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+{% include integrations/config_flow.md %}
+
+The Qingping integration will automatically discover devices once the [Bluetooth](/integrations/bluetooth) integration is enabled and functional.
+
+## Supported devices
+
+- [Air Monitor Lite](https://www.qingping.co/air-monitor-lite/overview) (CGDN1)
+- [Alarm Clock](https://www.qingping.co/bluetooth-alarm-clock/overview) (CGPR1)
+- [Motion & Ambient Light Sensor](https://www.qingping.co/motion-light-sensor/overview) (CGPR1)
```
Comment (MartinHjelmare) [inline source/_integrations/qingping.markdown:30]: Both the last two devices says CGPR1.

Comment (bdraco) [inline source/_integrations/qingping.markdown]:
```suggestion
- [Air Monitor Lite](https://www.qingping.co/air-monitor-lite/overview) (CGDN1)
- [Alarm Clock](https://www.qingping.co/bluetooth-alarm-clock/overview) (CGD1)
- [Motion & Ambient Light Sensor](https://www.qingping.co/motion-light-sensor/overview) (CGPR1)
```

Comment (bdraco) [reply source/_integrations/qingping.markdown:30]: Fixed

**Review by MartinHjelmare — APPROVED** [review]

**Comment by bdraco** [comment]
Thanks

**Comment by zachberger** [comment]
I tested this out today with my Qingping Air Monitor Lite. It didn't immediately work. I reset the sensor and set it up as QingPing protocol and then it showed up in hass. It had previously been setup in HomeKit mode. It may be worth adding that the device must not be paired with HomeKit for BLE to work

**Comment by bdraco** [comment]
I have mine paired with HomeKit and its working just fine with BLE as well.  There are screenshots in the core PR of them side by side.  I did have to update the firmware before the device was stable.

**Comment by zachberger** [comment]
that may be it, after resetting I found I had two firmware updates

**Pull request merged**